Assets/MixedRealityToolkit/Providers/BaseInputDeviceManager.cs

```diff
@@ -59,7 +59,7 @@ protected virtual IMixedRealityPointer[] RequestPointers(SupportedControllerType
         {
             var pointers = new List<IMixedRealityPointer>();
 
-            if ((Service != null) &&
+            if ((InputSystem != null) &&
```

Comment:
What does this change do?

Reply:
Service == InputSystem. It just makes this more consistent